torch/onnx/__init__.py

```diff
@@ -40,7 +40,7 @@ def set_training(model, mode):
             model.train(old_mode)
 
 
-def export(model, args, f, export_params=True, verbose=False, training=False):
+def export(model, args, f, input_names=[], output_names=[], export_params=True, verbose=False, training=False):
```

Comment:
Although input-names and output_names are not modified in our case,
I would suggest don't init the parameters with a mutable default value.

Reply:
Yes, it's better to allocate lists inside the function and make `None` the defaults. Also, it's a better idea to add arguments at the end. So far this call `torch.onnx.export(model, args, f, True)` was valid, but wouldn't be after this patch

Reply:
yeah, I forgot about this particular python feature for a moment. I'll change them to None, and at the end